<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет зарплаты</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,5,30,2">
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Операции</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Доплаты</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Вычеты</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Итого</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_all_operation">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_all_minus">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_all_sum">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_all_plus">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Дата закрытия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDateEdit" name="de_pay_date">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Расчитать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Закрыть месяц</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tw_main_salary">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>35</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>18</number>
     </attribute>
     <column>
      <property name="text">
       <string>Работник</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Итого</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Операции</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Доплаты</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Вычеты</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Операций</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Доплат</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Вычетов</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_calc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tw_main_salary</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>Dialog</receiver>
   <slot>ui_double_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>-27</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_save_salary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_calc()</slot>
  <slot>ui_double_click()</slot>
  <slot>ui_save_salary()</slot>
 </slots>
</ui>
